lib/bugsnag.js

```diff
@@ -116,6 +116,21 @@ Bugsnag.notify = function(error, options, cb) {
     if (!options) {
         options = {};
     }
+    if (!error && typeof error !== "string") {
+        Configuration.logger.error("Bugsnag.notify(error…) accepts an object or a string but was called with '" + error + "'");
+        options.metaData = {
+            "notify() arguments":
+                JSON.stringify(
+                    [].map.call(arguments, function(arg) { return arg }),
+                    function replacer(key, value) {
```

Comment:
I felt the need to implement this [custom replacer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#The_replacer_parameter) so that `undefined` and `Function` properties aren't misleadingly stripped/replaced with `null`.